.travis.yml

```diff
@@ -213,3 +218,17 @@ script:
 
   # Run regression tests (xtest in QEMU)
   - (cd ${HOME}/optee_repo/build && $make check CROSS_COMPILE="ccache arm-linux-gnueabihf-" AARCH32_CROSS_COMPILE=arm-linux-gnueabihf- CFG_TEE_CORE_DEBUG=y DUMP_LOGS_ON_ERROR=1)
+
+env:
+  global:
+    secure: "k30FG3hL6dQywtvMeEjSM0FQ7A0r+AQriUkGSFv7bmwD3EX2H17tJsfZt/h3dj+Pj6lQPdhVg3YO6O6pksJLqBLyz86bN15JEik0e1zftZplzK00oT89eoQr+qgqyrbs6CZBRvS98EAneCNbvgJto9sIPJs0SGu0u31Pr06bTAg="
```

Comment:
This is ok. All .travis.yml that use this addon have it, e.g. [mbedtls](https://github.com/ARMmbed/mbedtls/blob/development/.travis.yml#L26).

It's not some sort of key but rather an encrypted blob of the Coverity scan token required to upload results to Coverity's server.